### PR TITLE
#6: Added event handler skeleton

### LINKED ISSUE #6: Add Python Event-Handler Framework
# Background

- one of the parts of the Framework will be a python event handler. Usually, the logic of the event handler is implemented by the user, but we need to implement the Framework to execute this event handler.
- The Framework is responsible for the loading and storing the state, for call the user provided event handler and managing temporary tables
- To test event handler a python implementation of the event loop is useful, because debugging with the Lua event loop is probably difficult
- Users need to be able to specify the class which implements the event handler logic
  - Alternative, they might define a UDF from which this class is imported. This is for adhoc algorithms, when we don't want to build a python package or language container for the algorithm.

# Acceptance Criteria

- Create Python Event-Handler Framework
- Create Python Event Loop for Testing

### LINKED ISSUE #18: Prepare the skeleton of Event-Handler Framework
- add a temporary UDF stub
- add deployment scripts
- prepare test fixtures

### PR DESCRIPTION
**Pull request by umitbuyuksahin**
Fixes #18

### CODE REVIEW

File: exasol_advanced_analytics_framework/deployment/scripts_deployer_cli.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import click
+from exasol_advanced_analytics_framework.deployment import utils
+from exasol_advanced_analytics_framework.deployment.scripts_deployer import \
+    ScriptsDeployer
+
+
+@click.command(name="scripts")
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/deployment/scripts_deployer_cli.py:7]: oh this is new, didn't know this is possible

File: run_integration_tests.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Run unit tests
```
Comment (tkilias) [inline run_integration_tests.yaml:1]: this file is wrongly located, I guess

File: exasol_advanced_analytics_framework/interface/create_event_handler_udf.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+class CreateEventHandlerUDF:
+    def __init__(self, exa):
+        self.exa = exa
+
+    def run(self, ctx) -> None:
+        ctx.emit("return query")
```
[the remark this answers is not in the document]
Comment (umitbuyuksahin) [reply exasol_advanced_analytics_framework/interface/create_event_handler_udf.py:7]: yes, exactly.

File: tests/deployment/test_language_container_deployer.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import textwrap
+from exasol_advanced_analytics_framework.deployment.language_container_deployer \
+    import LanguageContainerDeployer
+from tests.utils.revert_language_settings import revert_language_settings
+from tests.utils.db_queries import DBQueries
+from tests.utils.parameters import bucketfs_params, db_params
+from pathlib import Path
+
+
+@revert_language_settings
```
[the remark this answers is not in the document]
Comment (umitbuyuksahin) [reply tests/deployment/test_language_container_deployer.py:10]: yes, this decorator function is only specific to where it is called.

**Pull request closed without merging**